### DISCUSSION TITLE
Newbies

### DISCUSSION THREAD

**Discussion by yeehi** [paveloom-f/lbry-desktop#10] [Ideas]
For technically unsophisticated users, people unfamiliar with git, in a prominent place, please  leave a few lines explaining what to do to get the binary they want.  
  
For example: "To download the file for GNU+Linux, Windows or Mac, click on the "Actions" tab above, and follow the top link for "Build and Release." Unzip/extract the downloaded file and then install it. "

I think you ought to mention to not let it update automatically, as that recreates the blacklist.

**Comment by paveloom** [comment]
> For example: "To download the file for GNU+Linux, Windows or Mac, click on the "Actions" tab above, and follow the top link for "Build and Release." Unzip/extract the downloaded file and then install it. "

Actually, one is supposed to take the binaries from the [Releases](https://github.com/paveloom-f/lbry-desktop/releases) page.

The description is extended now.

**Comment by paveloom** [comment]
Just a heads-up: since [`v0.51.1`](https://github.com/paveloom-f/lbry-desktop/releases/tag/v0.51.1), self-updating the client will not reset the blacklist anymore.
